core/baseline-core-v1.0-psd01.md

```diff
@@ -566,179 +566,631 @@ Below we list and define the components of each layer. The detailed requirements
 
-## 4.1 Authentication and Authorization
+As a result, companies spend significant resources fighting security breaches caused by CAs, and public internet communications that are both truly secure and user-friendly are still out of reach for most.
 
-Describes the requirements for the Authentication service responsible for establishing and maintaining the connection between verified parties.
+Given the above, the Baseline Protocol Standard identity approach is as follows: Every identity is controlled by its Principal owner and not by a 3rd party, unless the Principal Owner has delegated control to a 3rd party. 
 
-| Requirement ID | Requirement  | 
-| :--- | :--- |
-| COM1 | something MUST something |
-| COM2 | something SHOULD something |
+A Principal Owner is defined as the entity controlling the public key(s) which control the identity and its identifiers upon inception of the identity.
 
-## 4.2 Message Delivery
+The Baseline Protocol Standard defines identity in the context of this document to mean the following:
 
-Describes the requirements for the Delivery service responsible for exchanging messages between verified parties.
+```
+Identity = <Identifier(s)> + <associated data>
+```
+where associated data refers to data describing the characteristics of the identity that is associated with the identifier(s). An example of such associated could be an X.509 issues by a CA. 
 
-| Requirement ID|Requirement  |
-| :--- | :--- |
-| COM3 | MUST implement IMessagingService interface to interact with verified parties. |
-| COM4 | MUST implement pub-sub pattern. |
-| COM5 | MUST implement XYZ Protocol Messages (types, formats, queues, inbound, outbound). |
-| COM6 | Automated message validation |
-| COM7 | Messaging Endpoints - SHOULD be distinct from system of record. |
+This approach requires a decentralized, or at least strongly federated, infrastructure as expressed in the requirements below. 
 
-## 4.3 Performance
+**[D5]**	The Public Key Infrastructure (PKI)  of a BPI SHOULD be decentralized.
 
-Describes the performance requirements for the Communication component.
+Decentralized in this context means that there is no single point of failure in the PKI where possibly no participants are known to one another.
 
-| Requirement ID|Requirement  |
-| :--- | :--- |
-| COM8 | Durable messaging  |
-| COM9 | Persistent messaging|
-| COM10 | Latency  |
-| COM11 | Fault-tolerance |
-| COM12 | Scalability |
+**[R33]**	The PKI of a BPI MUST be strongly federated.
```

Comment:
Any example or concrete number to be strongly federated?

Reply:
updated text to contextualize without giving hard number